utils/dataconverter.go

```diff
@@ -274,6 +276,24 @@ func (mS *DurationConverter) Convert(in any) (
 	return IfaceAsDuration(in)
 }
 
+func NewDurationFormatConverter(params string) (hdlr DataConverter, err error) {
+	return new(DurationFormatConverter), nil
+}
+
+// DurationFormatConverter formats duration to hh:mm:ss
```

Comment:
I think a better explanation should be: formats duration in the same way Time is formatted as string.